pandas/core/reshape/concat.py

```diff
@@ -532,6 +531,7 @@ def _get_comb_axis(self, i: int) -> Index:
             copy=self.copy,
         )
 
+    @cache_readonly
     def _get_concat_axis(self) -> Index:
```

Comment:
If it is property, then maybe ``_concat_axis``, without ``get`` would be better?